curl/bct/transform_amd64.s

```diff
@@ -0,0 +1,76 @@
+// +build amd64,!gccgo,!appengine
```

Comment:
The `bct.Curl` assembler version provides a speed-up of about 10%. Does this speed-up justify the introduction of assembler code which is much harder to review and maintain?

Reply:
Since you've already also introduced assembler code for Curl I don't see any reason why it then should be left out for bct.Curl. (Maintenance complexity stays the same)